sentry_sdk/utils_py3.py

```diff
@@ -0,0 +1,31 @@
+from sentry_sdk.hub import Hub
+from functools import wraps
+
+from sentry_sdk._types import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from collections.abc import Callable
+    from typing import Optional
+    from sentry_sdk.integrations import Integration
+
+    from sentry_sdk._types import GenericAsyncCallable
+
+
+def integration_patched_async(original_function, integration=None):
```

Comment:
I would called it `ensure_integration_enabled(integration=None, original_function=None)` to make it easier to understand what is happening. (And if `original_function` is None, just fail with an error)